src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java

```diff
@@ -224,9 +225,27 @@ public static String getPcreportFileName() {
         return pcReportFileName;
     }
 
+    private String getVersion() {
+        try {
+            String completeVersion = ConfigurationService.getPluginVersion();
+            if(completeVersion != null) {
+                String[] partsOfCompleteVersion = completeVersion.split(" [(]");
+                if (partsOfCompleteVersion != null)
+                    return partsOfCompleteVersion[0];
+            }
+        }
+        catch(Exception e)
```

Comment:
Please use only the exception that is being thrown and not swallowing all the unreleased exceptions, split throws PatternSyntaxException and please log the issues as well.